wrench/src/yaml_frame_reader.rs

```diff
@@ -616,23 +659,23 @@ impl YamlFrameReader {
             if let Some(clip_scroll_info) = clip_scroll_info {
                 dl.push_clip_and_scroll_info(clip_scroll_info);
             }
-
             let local_clip = self.get_local_clip_for_item(item, full_clip);
+            let backface_visible = item["backface-visible"].as_bool().unwrap_or(true);
             match item_type {
-                "rect" => self.handle_rect(dl, item, local_clip),
-                "line" => self.handle_line(dl, item, local_clip),
-                "image" => self.handle_image(dl, wrench, item, local_clip),
-                "text" | "glyphs" => self.handle_text(dl, wrench, item, local_clip),
+                "rect" => self.handle_rect(dl, item, local_clip, backface_visible),
```

Comment:
we could probably make all of those handlers to receive `LayoutPrimitiveInfo` with zeroed out rectangle, for simplicity